src/FSLibrary/StellarKubeSpecs.fs

```diff
@@ -692,6 +692,7 @@ type NetworkCfg with
         let initCommands = self.getInitCommands cfgOpt coreSet.options
 
         let runCmd = [| "run" |]
+        let runCmd = Array.append runCmd [| "--console" |]
```

Comment:
I think this commit can be dropped now that we merged https://github.com/stellar/supercluster/pull/62. Github checks should also be fixed now (we had a super old image hardcoded there, so it was failing to parse the new flag)